terraform/aks/application.tf

```diff
@@ -16,7 +16,10 @@ module "application_configuration" {
   is_rails_application = true
 
   config_variables = {
-    PGSSLMODE        = local.postgres_ssl_mode
+    PGSSLMODE           = local.postgres_ssl_mode
+    BIGQUERY_DATASET    = var.enable_dfe_analytics_federated_auth ? module.dfe_analytics[0].bigquery_dataset : null
```

Comment:
This doesn't matter if federated auth is enabled or not, as BIGQUERY_DATASET, BIGQUERY_PROJECT_ID and BIGQUERY_TABLE_NAME won't change

Reply:
Updated